modal/runner/engines/vllm.py

```diff
@@ -89,7 +90,7 @@ async def generate(self, payload: Payload, params):
                 async for request_output in results_generator:
                     final_output = request_output
 
-                output = request_output.outputs[0].text
```

Comment:
probs worth reading this specific change a bit closer

only glanced at it, but had a possible scoping bug where request_output was being accessed after the loop terminates

think behavior should be the same with this

Reply:
Good catch - will run the e2e test without stream on this path, but this diff makes sense to me